src/features/game/players/Players.tsx

```diff
@@ -1,31 +1,59 @@
-import React from "react";
+import React, { ReactElement } from "react";
 import { useSelector } from "react-redux";
 import {
   getActiveGamePlayers,
   getActiveGameSpectator,
   getActiveGameTurn,
 } from "../../../redux/games";
+import { getCurrentPlayerId } from "../../../redux/currentPlayer";
 import ActivePlayer from "./ActivePlayer";
 import Spectator from "./Spectator";
 
 const Players = () => {
   const players = useSelector(getActiveGamePlayers);
   const spectators = useSelector(getActiveGameSpectator);
+  const currentPlayer: string = useSelector(getCurrentPlayerId);
   const turn = useSelector(getActiveGameTurn);
+
+  const currentPlayerCard: ReactElement[] = [];
+  const restPlayers: ReactElement[] = [];
+
+  (() => {
+    players.forEach((each: string) => {
```

Comment:
Lets avoid usage of forEach and prefer filter and map instead.

Also why do we need to loop for the current play tile? Can we just do the following?

```typescript
<ActivePlayer
  key={currentPlayer}
  playerId={currentPlayer}
  players={players}
  turn={turn}
/>
```